sysutils/go2port/Portfile

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            amake go2port cb963704f0690cb44d67da6184206bcbabb20f1f
```

Comment:
BTW, this could also be hosted as part of the MacPorts git repo. We have some other such tools in the tree.

Reply:
This tool is written in Go, so I thought it made more sense to be a separate project. (Also I wanted to dogfood the portgroup.)